components/Tag/Tag.jsx

```diff
@@ -38,37 +20,50 @@ class Tag extends Component {
 
   constructor() {
     super();
+
     this.state = {
-      isHover: false,
-      isRemoveHover: false,
-    };
-    this.setHover = this.setHover.bind(this);
-    this.setRemoveHover = this.setRemoveHover.bind(this);
+      isHovering:false,
+      isHoveringOverX:false
+    }
+
+    this.handleRemove = this.handleRemove.bind(this);
+    this.getXColor = this.getXColor.bind(this);
   }
 
-  setHover(value) {
-    return () => this.setState({ isHover: value });
+  handleRemove (event) {
+
+    console.log('click')
+    if(this.props.onRemove) {
```

Comment:
Things like this should be caught by the linter, but due to https://github.com/vhx/quartz-react/pull/48 they were not. Might be a good idea to pull and merge master, then run `npm run lint`?